jbpm-bom/pom.xml

```diff
@@ -899,6 +899,19 @@
         <classifier>sources</classifier>
       </dependency>
 
+      <!-- service task repository -->
+      <dependency>
+        <groupId>org.jbpm.contrib</groupId>
+	      <artifactId>repository</artifactId>
```

Comment:
@mswiderski formatting looks a bit off here, would you mind double checking...

Reply:
@cristianonicolai adjusted formatting